packages/extra/bootbox/templates/matchbox/deployment.yaml

```
@@ -0,0 +1,54 @@
+    metadata:
+      labels:
+        app: bootbox-matchbox
+    spec:
+      containers:
+      - name: matchbox
+        image: "{{ $.Files.Get "images/matchbox.tag" | trim }}"
+        args:
+          - "-address=:8080"
+          - "-log-level=debug"
+        volumeMounts:
+          - name: profiles-volume
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)